addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1174,6 +1160,67 @@ function _applyColspan(element, colspan, tableWidth) {
     element.style.setProperty('max-width', width + 'px');
     element.classList.toggle('o_converted_col', true);
 }
+/**
+ * Take an element with a background image and return a string containing the
+ * VML code to display the same image properly in Outlook, with its contents
+ * inside.
+ * TODO: We assume:
+ *      - background-size: cover,
+ *      - background-repeat: no-repeat,
+ *      - size 100%
+ *      - content is centered x/y
+ * These assumptions may not hold for cover snippet and other uses. This needs
+ * to account for that.
+ * TODO: centering span probably not needed with `v-text-anchor:middle` present.
```

Comment:
Are these TODO still relevant after all your tests and fixes ? Just wondering.

Reply:
Done updating the comment.